quickstart/101-ai-studio/dependent.tf

```diff
@@ -0,0 +1,66 @@
+
+resource "azurerm_resource_group" "default" {
+  name     = "rg-${var.names}"
+  location = var.location
+}
+
+resource "azurerm_storage_account" "default" {
+  name                            = "${var.names}storage"
+  location                        = azurerm_resource_group.default.location
+  resource_group_name             = azurerm_resource_group.default.name
+  account_tier                    = "Standard"
+  account_replication_type        = "GRS"
+  allow_nested_items_to_be_public = false
+}
+
+resource "azurerm_key_vault" "default" {
+  name                     = "${var.names}keyvault"
```

Comment:
We could add a random string suffix here to avoid potential naming conflict:

```hcl
resource "random_string" "suffix" {
  length           = 4
  special          = false
  upper            = false
}

resource "azurerm_key_vault" "default" {
  name                     = "${var.names}keyvault${random_string.suffix.result}"
  location                 = azurerm_resource_group.default.location
  resource_group_name      = azurerm_resource_group.default.name
  tenant_id                = data.azurerm_client_config.current.tenant_id
  sku_name                 = "standard"
  purge_protection_enabled = false
}
```

Reply:
I included this. Hopefully the e2e tests run successfully.